server/schedule/operator_controller.go

```diff
@@ -973,3 +972,30 @@ func (oc *OperatorController) RemoveStoreLimit(storeID uint64) {
 	}
 	delete(oc.storesLimit, storeID)
 }
+
+// CollectStoreLimitMetrics collects the metrics about store limit
+func (oc *OperatorController) CollectStoreLimitMetrics() {
+	oc.RLock()
+	defer oc.RUnlock()
+	if oc.storesLimit == nil {
+		return
+	}
+	stores := oc.cluster.GetStores()
+	for _, store := range stores {
+		if store != nil {
+			storeID := store.GetID()
+			if oc.storesLimit[storeID] == nil {
+				continue
+			}
+			for n, v := range storelimit.TypeNameValue {
+				storeLimit := oc.storesLimit[storeID][v]
+				if storeLimit == nil {
+					continue
+				}
+				storeIDStr := strconv.FormatUint(storeID, 10)
+				storeLimitAvailableGauge.WithLabelValues(storeIDStr, n).Set(float64(storeLimit.Available()) / float64(storelimit.RegionInfluence[v]))
+				storeLimitRateGauge.WithLabelValues(storeIDStr, n).Set(storeLimit.Rate() * StoreBalanceBaseTime)
```

Comment:
Users may more likely want to know the store how many operators can be produced in one minute, for example, default is 15 opm.

Reply:
The metrics `storeLimitRateGauge` can show the operator producing rate.

Reply:
got.